it/uk/gov/hmrc/soletraderidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -624,6 +666,46 @@ class CheckYourAnswersControllerISpec extends ComponentSpecHelper
         verifyStoreIdentifiersMatch(testJourneyId, JsString(SuccessfulMatchKey))
         verifyAuditTypeFor(auditTypeToBeFound = "IndividualIdentification")
       }
+      "the provided details match what is held in the database (nino is fraudulent)" in {
+        enable(EnableNoNinoJourney)
+        await(journeyConfigRepository.insertJourneyConfig(
+          journeyId = testJourneyId,
+          authInternalId = testInternalId,
+          journeyConfig = testIndividualJourneyConfig
+        ))
+        stubAuth(OK, successfulAuthResponse())
+        stubRetrieveIndividualDetails(testJourneyId)(OK, testIndividualDetailsJsonNoSautr).setNewScenarioState("auditing")
+        stubMatch(testIndividualDetailsNoSautr)(OK, successfulMatchJson(testIndividualDetailsNoSautr))
+        stubIsFraudulentNino(testNino)(OK, fraudulentNinoJson)
+        stubStoreIdentifiersMatch(testJourneyId, NinoIsFraudulent)(OK)
+        stubStoreBusinessVerificationStatus(testJourneyId, BusinessVerificationNotEnoughInformationToCallBV)(OK)
+        stubStoreRegistrationStatus(testJourneyId, RegistrationNotCalled)(OK)
+
+        stubAudit()
+        stubRetrieveIndividualDetails(testJourneyId)(OK, testIndividualDetailsJsonNoSautr ++ Json.obj("identifiersMatch" -> NinoIsFraudulentKey)).setRequiredScenarioState("auditing")
+        stubRetrieveFullName(testJourneyId)(OK, Json.toJsObject(FullName(testFirstName, testLastName)))
+        stubRetrieveDob(testJourneyId)(OK, Json.toJson(testDateOfBirth))
+        stubRetrieveNino(testJourneyId)(OK, testNino)
+        stubRetrieveSautr(testJourneyId)(NOT_FOUND)
+        stubRetrieveIdentifiersMatch(testJourneyId)(OK, NinoIsFraudulent)
+        stubRetrieveAuthenticatorDetails(testJourneyId)(NOT_FOUND)
+        stubRetrieveBusinessVerificationStatus(testJourneyId)(NOT_FOUND)
+        stubRetrieveRegistrationStatus(testJourneyId)(NOT_FOUND)
+        stubRetrieveAuthenticatorFailureResponse(testJourneyId)(OK, "NinoNotFound")
+
```

Comment:
Are all these retrieves for the audit? Looks like it only retrieves the individual details,  identifiersMatch and authenticator response. We're not storing anything for the authenticator response so it can return NOT FOUND

Reply:
Yes its for the audit! 😊